scan

```diff
@@ -80,13 +80,15 @@ def scan_domains(scanners, domains):
             'writer': scanner_writer
         }
 
-    # For each domain, run through each scanner.
-    for domain in domains_from(domains):
-        for scanner in scanners:
-            # A scanner can return multiple rows.
+    # A scanner can return multiple rows.
+    for scanner in scanners:
+        # For each domain, run through each scanner.
+        for domain in domains_from(domains):
             for row in scanner.scan(domain, options):
                 if row:
                     handles[scanner]['writer'].writerow([domain] + row)
+        if scanner.__name__ == 'scanners.googleanalytics':
+            scanner.clean_up()
```

Comment:
Instead of hardcoding in an exception here, the scanning process should look to see if a scanner defines a `cleanup()` method, and if it does, execute it.

However, if we follow my above note and restore the scanning loop, this scanner won't need to hold open a single browser instance anymore.